ItemDescription3/pageEXT/itemCardPagExt.50400.PageExt.al

```diff
@@ -0,0 +1,16 @@
+pageextension 50400 appitemCardPagExt extends "Item Card"
+{
+    layout
+    {
+        addafter("No.")
```

Comment:
I didn't specify where exactly description should be located. But it's for sure should not be after No.
Usually best practice is to by default to use "AddLast" or "AddFirst" to specific group. For example in this case better place will be AddLast in repeater of all fields.

It's also ok to add such field after after description 2 for example